frontends/torch-frontend/torch-frontend/lib/Conversion/ConvertTorchToCcl.cpp

```diff
@@ -245,7 +245,7 @@ class ConvertSendOp : public OpConversionPattern<OP> {
     }
 
     auto cclSendOp = rewriter.create<ccl::SendOp>(
-        op->getLoc(), outType, input,
+        op->getLoc(), outType, input, Value(),
```

Comment:
Now it seems that the collective operators we traced from the PyTorch program are always static, which means we will always use I64Attr instead of the I64 operand in the conversion. How about we directly use the I64 operand?

Reply:
It's not necessary to involve I64 operand between frontends and compiler. it will bring some dialects expect stablehlo, like arith dialect.

Reply:
The `dynamic_target_index` should only used by compiler's transform passes.